modules/ROOT/pages/custom-authorization-rule-mapping.adoc

```diff
@@ -37,7 +37,7 @@ In this example, Custom authorization rule is a maven-based java project that ne
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
**thought:** I have just discovered that we have the issue only when `<?xml version="1.0" encoding="UTF-8"?>` is present before the line where we see the issue.
I have detected this after you revert a change including the workaround that is not working: https://github.com/bonitasoft/bonita-doc/pull/2324#pullrequestreview-1301658283

Did you try to remove the XML directive to see if this doesn't fix the problem without the need of the workaround?

Reply:
Yes, that was my motivation, but it did not seem to be enough to get rid of the problem.

Reply:
OK, fine. 😸